doc/developer-guide.md

```diff
@@ -319,23 +319,28 @@ The builtin CMake options used when invoking CMake on the command line.
   `-DCMAKE_SKIP_RPATH=ON` when configuring CMake in build directory.
 
 * `CA_HOST_TARGET_<arch>_CPU`: This option is used by the `host` target to
-  optimize for performance on a given CPU.`arch` should be a capitalized
+  optimize for performance on a given CPU. `arch` should be a capitalized
   version of the `host` target architecture e.g. `X86_64`, `RISCV64` or
   `AARCH64`. If set to "native" host will optimize for the CPU being used to
   compile it. Otherwise a CPU name can be provided, for example "skylake", but
   be warned that this string will be passed directly to the llvm backend so make
   sure it's a valid CPU name. Information about your host CPU can be found by
-  running`llc --version`, and a list of host CPUs supported by your installed
-  version of LLVM can be found by running`llc --march=[your-arch] --mcpu=help`.
+  running `llc --version`, and a list of host CPUs supported by your installed
+  version of LLVM can be found by running `llc --march=[your-arch] --mcpu=help`.
+
+  Setting a specific `CA_HOST_TARGET_<arch>_CPU` ignores the default features
+  and enables the features supported by that CPU.
   
   Be aware that if `host` is compiled with this option set, running it on a
   different CPU from the one specified (or the one compiled with if "native" was
-  specified) isn't supported and bad things may happen. When the oneAPI
-  Construction Kit is built in debug mode, the environment variable
-  `CA_HOST_TARGET_CPU` is also respected across all `host` targets, which can
-  help track down codegen differences among different machine targets. The
-  caveats above apply, and this may result in an illegal instruction crash if
-  your CPU doesn't support the generated instructions.
+  specified) isn't supported and bad things may happen.
+
+* `CA_HOST_TARGET_<arch>_FEATURES`: This option is used by the `host` target to
```

Comment:
I think it's worth saying that each architecture will have default features which is extended unless the CPU has been overriden and the env variable will extend any features defined in this cmake.